packages/eas-cli/src/commands/update/configure.ts

```diff
@@ -54,21 +54,11 @@ export default class UpdateConfigure extends EasCommand {
     Log.addNewLineIfNone();
     Log.log(`🎉 Your app is configured with EAS Update!`);
     Log.newLine();
-    Log.log(`${chalk.bold('Next steps')}:`);
-    Log.newLine();
-    Log.log('Update a production build:');
-    Log.log(`1. Create a new build. Example: ${chalk.bold('eas build --profile production')}.`);
-    Log.log('2. Make changes in your project.');
-    Log.log(`3. Publish an update. Example: ${chalk.bold('eas update --channel production')}.`);
-    Log.log('4. Force close and reopen the app at least twice to view the update.');
-
-    Log.newLine();
-    Log.log('Preview an update:');
-    Log.log(
-      `1. Publish an update to a branch. Example: ${chalk.bold('eas update --branch new-feature')}.`
-    );
+    Log.log(`- Run ${chalk.bold('eas build:configure')} to complete your installation`);
```

Comment:
if they've already run `eas build:configure`, does this message seems lead them astray?